services/core-api/app/api/constants.py

```diff
@@ -83,6 +83,9 @@ def DOWNLOAD_TOKEN(token_guid):
     return f'document-manager:download-token:{token_guid}'
 
 
+#Deep Update Special Flag
+DELETE_ITEM_FROM_LIST_JSON_KEY = "DELETE_ON_PUT"
```

Comment:
After discussion let's make this a more generic state name that includes delete.